src/components/App/App.js

```diff
@@ -56,7 +62,12 @@ export default class App extends Component {
     return (
       <div className="App">
         <Switch>
-          <Route path='/areas' render={ () => <Areas areaSpecs={this.state.areaDetails} /> } />
+          <Route path='/areas/:id' render={ () => <Listings 
```

Comment:
Make sure you set up an `.eslintrc` file. The formatting here looks a little off.